<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageStandingOrders</class>
 <widget class="QDialog" name="ManageStandingOrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Daueraufträge verwalten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableStandingOrders">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Konto</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Typ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Wert</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nächste Ausführung</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kategorie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Beschreibung</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonAddOrder">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Dauerauftrag hinzufügen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonChangeOrder">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Dauerauftrag ändern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDeleteOrder">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Dauerauftrag löschen</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBack">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Zurück</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>ManageStandingOrders</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>887</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>940</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
